src/store/ComparisonStore.js

```diff
@@ -120,4 +139,52 @@ export default class ComparisonStore {
 
         return selectedColor;
     }
+
+    @action.bound onSymbolChange({ action : symbolAction }) {
+        if (symbolAction === 'master') {
+            const { stx } = this.context;
+            this.comparisonSymbols = [];
```

Comment:
@brucebinary If internet stays offline for a wile and the go back online the compariosn list will be cleared because the chart is being refreshed and `onSymbolChange` will be raised, i suppose.